crossdata-driver/src/main/scala/com/stratio/crossdata/driver/Driver.scala

```diff
@@ -44,14 +44,17 @@ class Driver(val seedNodes: java.util.List[String] = new util.ArrayList[String](
     case _ => Driver.config
   }
 
-  private val system = ActorSystem("CrossdataSystem", finalConfig)
+  private val system = ActorSystem("CrossdataServerCluster", finalConfig)
 
   if(logger.isDebugEnabled){
     system.logConfiguration()
   }
 
-  private val contactPoints = finalConfig.getStringList("akka.cluster.seed-nodes")
+  private val contactPoints = finalConfig.getStringList("akka.cluster.seed-nodes").map(dir=>dir+"/user/receptionist")
+
   private val initialContacts: Set[ActorSelection] = contactPoints.map(contact => system.actorSelection(contact)).toSet
+  println("Initial contacts: " + initialContacts)
```

Comment:
Println must be avoided, use the logger instead if applied. Otherwise, it should be removed